src/main/src/main/java/org/geoserver/security/GeoServerSecurityManager.java

```diff
@@ -2687,7 +2758,14 @@ public boolean accept(File pathname) {
     }
 
     XStreamPersister globalPersister() throws IOException {
-        XStreamPersister xp = persister();
+        if (gxp == null) {
```

Comment:
Should not be necessary, but some tests are not doing proper initialization, so we make sure to check.